backend/Squirrel.Shared/Exceptions/UserCannotBeRemovedException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Squirrel.Shared.Enums;
+using Squirrel.Shared.Exceptions.Abstract;
+using System.Net;
+
+namespace Squirrel.Shared.Exceptions;
+
+public sealed class UserCannotBeRemovedException : RequestException
+{
+    public UserCannotBeRemovedException() : base(
+        "User can not be removed.",
+        ErrorType.UserCannotBeRemoved,
+        HttpStatusCode.BadRequest)
```

Comment:
You only use this exception if the user does not have a permission to remove users from a project, so rename the exception to something like "InvalidPermissionsException" and change HttpStatusCode to Forbidden.